### ISSUE TITLE
Output models are not saved

### ISSUE THREAD

**Issue by sajastu** [OpenNMT/OpenNMT-py#1353]
Hi,

I'm trying to train a Transformer model. it seems that the training proceeds, however, there is no output model coming out into the /models folder.

This is the command I'm using for the training:

`python train.py -save_model models/cnn/CNN -data data/cnn/CNN -word_vec_size 100  -encoder_type transformer -decoder_type transformer -pre_word_vecs_enc "data/cnn/embeddings.enc.pt" -pre_word_vecs_dec "data/cnn/embeddings.dec.pt" -train_steps 200000 -dropout 0.5 -batch_size 64 -optim adagrad -learning_rate 0.001 -seed 1234 -transformer_ff 2048 -heads 5 -position_encoding -share_embeddings -adam_beta2 0.998 -label_smoothing 0.1 -decay_method noam  -warmup_steps 8000 -rnn_size 100 -world_size 1 -gpu_ranks 1`

The `/cnn` folder is created under models directory, there are no output models there (inside cnn fokder) though.

**Comment by sajastu** [comment]
I'm not sure where the problem exactly came from. But I just changed `-save_model models/cnn/CNN` to simply `-save_model models/cnn` and then the program could finally output saved models to `/models` directory.
